src/CLMWrapper.cpp

```diff
@@ -508,7 +509,10 @@ void ClmWrapper::callback(const sensor_msgs::ImageConstPtr& msgIn)
 
     });
 
-    headsPublisher.publish( ros_heads_msg );
+    //headsPublisher.publish( ros_heads_msg );
```

Comment:
why did you comment this out? The old publisher should still work even if you have the new one (they are doing different things).